tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -1133,7 +1197,7 @@ public async Task IsUpToDateAsync_True_AdditionalDependentFileTimeSourceIsOlderT
             var itemChangeTime = DateTime.UtcNow.AddMinutes(-4);
             var dependentTime = DateTime.UtcNow.AddMinutes(-3);
             var outputTime = DateTime.UtcNow.AddMinutes(-2);
-            var lastCheckTime = DateTime.UtcNow.AddMinutes(-1);
+            var lastCheckTime = DateTime.UtcNow.AddMinutes(3);
```

Comment:
This means the last check is in the future, which is impossible.

I don't think you should be touching the last check time in the implementation though, as commented above.